### PR TITLE
ci: switched from Travis-CI to GH Actions

### PR DESCRIPTION
**Pull request by travi**
-----
[View rendered README.md](https://github.com/travi/settings/blob/gh-actions/README.md)

### CODE REVIEW

**Comment by travi** [comment]
looks like we'll also need to switch the required check from travis to the new action

**Comment by travi** [comment]
i guess i havent done all that much with actions for ci yet. does the action need to make it to mainline before it would run for a PR, or do i simply have it configured wrong for that part?

File: .github/workflows/node-ci.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - uses: bahmutov/npm-install@v1
```
Comment (gr2m) [inline .github/workflows/node-ci.yml:20]: cool, I didn't know about this one :)

File: README.md [hunk 2]
```diff
@@ -167,3 +167,7 @@ See [docs/deploy.md](docs/deploy.md) if you would like to run your own instance
 [dependabot-link]: https://dependabot.com/
 
 [dependabot-badge]: https://badgen.net/dependabot/probot/settings/?icon=dependabot
+
+[github-actions-ci-link]: https://github.com/probot/settings/actions?query=workflow%3A%22Node.js+CI%22
```
Comment (gr2m) [inline README.md:171]: I would filter by master branch

```suggestion
[github-actions-ci-link]: https://github.com/probot/settings/actions?query=workflow%3A%22Node.js+CI%22+branch%3Amaster
```

File: .github/workflows/node-ci.yml (hunk 1 above)
Comment (travi) [reply .github/workflows/node-ci.yml:20]: when in doubt, its often worth checking to see if gleb has a solution :) very often pays off

File: README.md (hunk 2 above)
Comment (travi) [reply README.md]: good call. i also updated my scaffolder: https://github.com/form8ion/github-actions-node-ci/commit/eaec1bad8d8d3dcf757f4758764643fba85be6db

**Comment by travi** [comment]
thanks, @gr2m. i rebased the other PR and confirmed that the action does trigger there. are you able to mark that one as a required check like travis was?

**Pull request merged**